File: samples/snippets/fsharp/VS_Snippets_CLR_System/system.Console.WriteLine/FS/writeline_obj1.fs
module writeline_obj1

open System

// <Snippet3>
let values: obj [] = 
    [| true; 12.632; 17908; "stringValue"; 'a'; 16907.32M |]

for value in values do
   Console.WriteLine value

// The example displays the following output:
//    True
//    12.632
//    17908
//    stringValue
//    a
//    16907.32
// </Snippet3>

File: snippets/csharp/System/Console/WriteLine/writeline_obj1.cs
using System;

public class WriteLineObjectExample
{
    public static void Run()
    {
        // <Snippet3>
        object[] values = { true, 12.632, 17908, "stringValue",
                                 'a', 16907.32m };
        foreach (object value in values)
            Console.WriteLine(value);

        // The example displays the following output:
        //    True
        //    12.632
        //    17908
        //    stringValue
        //    a
        //    16907.32
        // </Snippet3>
    }
}
